gradle.properties

```diff
@@ -1,4 +1,4 @@
 group                  =  com.tailf.jnc
 archivesBaseName       =  project.name
-version                =  1.0
+version                =  1.0.1
 author                 =  Tail-f Systems
```

Comment:
Related to my comment on README.md, when should we bump the version number?

Reply:
Ok, in that case i suggest we use 1.1.0-SNAPSHOT

https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN401

Reply:
Fine with me.